### ISSUE TITLE
[SDK] Better Documentation

### ISSUE THREAD

**Issue by rbrisita** [thirdweb-dev/js#48]
The documentation lacks information on which methods work with which contract type. Reviewing method names (collection vs edition and various drops) and consolidating them with function overloading would be beneficial. There could also be an auto detection step as contract types are identifiable.

## No mention that it only works with ERC721
https://portal.thirdweb.com/pre-built-contracts/nft-collection
https://portal.thirdweb.com/sdk/interacting-with-contracts/nft-collection

## No mention that it only works with ERC1155
https://portal.thirdweb.com/pre-built-contracts/edition
https://portal.thirdweb.com/sdk/interacting-with-contracts/edition

Finally, labeling which methods contain a cost would also be helpful to developers.

**Comment by jarrodwatts** [comment]
Thanks for the feedback @rbrisita. We're actually working on breaking up our docs into EIP standards currently over in the docs repo. https://github.com/thirdweb-dev/docs

**Comment by SamSin369** [comment]
Currently out of the box downloading the starter project does not work for me in vite or react, I tried downgrading my react scripts to v 4.0.3 still getting errors
`ERROR in ./node_modules/@zag-js/react/dist/index.mjs 6:0-71
Module not found: Error: Can't resolve 'use-sync-external-store/shim' in '/home/sam/Desktop/help/node_modules/@zag-js/react/dist'
Did you mean 'index.js'?
BREAKING CHANGE: The request 'use-sync-external-store/shim' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
`
Using node LTS not sure what the issue is here
